clay/claycli
# Issue 130: Add github action for the doc's deploy process

Adding a `Github Action` to make the build process for the docs

Issue: https://github.com/clay/claycli/issues/130

# Steps for testing the site locally:
- Install [act](https://github.com/nektos/act#installation)
- Go to the [`.github` ](https://github.com/clay/claycli/tree/git_action/.github) folder
- On the `main.workflow` update the[` branch`](https://github.com/clay/claycli/blob/git_action/.github/main.workflow?short_path=8c5ae9c#L7) line to use `branch git_action`
- Use the [`act`](https://github.com/nektos/act#commands) command to simulate a `push` event. The build should be `successful`
- On the `main.workflow` update the[` branch`](https://github.com/clay/claycli/blob/git_action/.github/main.workflow?short_path=8c5ae9c#L7) line to use `branch master`
- Use the [`act`](https://github.com/nektos/act#commands) command to simulate a `push` event.  The build should be `decline`

Note:
Still missing the `github` token to complete the deploy process
